index.js

```diff
@@ -60,8 +62,8 @@ const apiGateway = axios.create({
     timeout: data.config.timeout,
 });
 
-Promise.all(data.messages.map(message => {
+for (const message of data.messages) {
     apiGateway.post('/DSTU2/$process-message', message)
-    .then(r => console.log('Success'))
-    .catch(e => console.error(e.message));
-}));
+        .then(r => console.log('Success'))
+        .catch(e => console.error(e.message));
```

Comment:
Sorry just one more thing, could you possibly make these log statements include the name of the file? I just feel like that would be a little more helpful. Other than that this looks good and I'm good to approve it!

Reply:
Yes! Done.